File: client/src/app/_models/financialReport.ts
export interface FinancialReport {
    id?: number;
    month: number;
    year: number;
    totalIncome: number;
    totalOutcome: number;
    netIncome: number;
    waterBill: number;
    electricityBill: number;
    gasBill: number;
    mobileBill: number;
    internetBill: number;
    insuranceBill: number;
    owedWaterBill: number;
}

File: API/Entities/FinancialReport.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace API.Entities
{
    [Table("FinancialReport")]
    public class FinancialReport
    {
        public int Id { get; set; }
        public int Month { get; set; }
        public int Year { get; set; }
        public int TotalIncome { get; set; }
        public int TotalOutcome { get; set; }
        public int NetIncome { get; set; }
        public int WaterBill { get; set; }
        public int ElectricityBill { get; set; }
        public int GasBill { get; set; }
        public int MobileBill { get; set; }
        public int InternetBill { get; set; }
        public int InsuranceBill { get; set; }
        public int OwedWaterBill { get; set; }
    }
}